nx.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "npmScope": "fxa",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": ["build", "lint", "test", "e2e"]
```

Comment:
Add test-unit and test-integration? These are targets used by CI and likely by developers too.

Reply:
Done